app/Resources/static/themes/material/index.js

```diff
@@ -120,4 +120,30 @@ $(document).ready(() => {
       });
     });
   }
+
+  $('.markasread').on('click', () => {
+    const article = document.getElementById('article');
+    const link = document.getElementById('link-archive');
+    const articleId = article.dataset.id;
+    const xhr = new XMLHttpRequest();
+
+    xhr.onload = function toggleArchive() {
+      if (xhr.status === 200) {
```

Comment:
Is there an icon indicating that a request is in progress? What if the request fails?